### DISCUSSION TITLE
how to disable this option in gopls

### DISCUSSION THREAD

**Discussion by explesy** [LazyVim/LazyVim#4588] [Q&A] [answered]
Can someone help me to understand how to disable all this things that I marked here with arrows
![image](https://github.com/user-attachments/assets/62f06bdb-b26c-43ce-b6a6-81b51b6bd788)

I believe it should be somewhere in gopls but I have no idea what the name of this option

And maybe this thing should be switched off by default?

**Comment by dpetka2001** [comment, marked as answer]
#3191 is what you want if I'm not mistaken.
